editor/store/actions.js

```diff
@@ -39,18 +39,20 @@ export function resetPost( post ) {
 /**
  * Returns an action object used to setup the editor state when first opening an editor.
  *
- * @param {Object} post   Post object.
- * @param {Array}  blocks Array of blocks.
- * @param {Object} edits  Initial edited attributes object.
+ * @param {Object}  post            Post object.
+ * @param {Array}   blocks          Array of blocks.
+ * @param {Object}  edits           Initial edited attributes object.
+ * @param {boolean} isValidTemplate Whether the template is valid or not.
  *
  * @return {Object} Action object.
  */
-export function setupEditorState( post, blocks, edits ) {
+export function setupEditorState( post, blocks, edits, isValidTemplate ) {
```

Comment:
I'm not sure this needs to be part of `setupEditorState`, particularly because it's not part of `state.editor` like the others are (and notably the others needing to be bulked together for undo/dirtying reasons).